include/boost/compute/memory_object.hpp

```diff
@@ -16,6 +16,7 @@
 #include <boost/compute/kernel.hpp>
 #include <boost/compute/detail/get_object_info.hpp>
 #include <boost/compute/detail/assert_cl_success.hpp>
+#include <boost/bind.hpp>
```

Comment:
Yeah it's not necessary. I don't know why I put this there, you're right that this belongs to the test.